nextflow_schema.json

```diff
@@ -788,6 +788,36 @@
                     "description": "Dataset scale for MetaBinner",
                     "help_text": "Dataset scale; eg. small,large,huge (default:large); Users can choose `huge` to run MetaBinner on huge datasets with lower memory requirements."
                 },
+                "skip_semibin": {
+                    "type": "boolean",
+                    "description": "Skip SemiBin2 Binning"
+                },
+                "semibin_rng_seed": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "RNG seed for SemiBin2.",
+                    "help_text": "SemiBin2 is run by default with a fixed seed within this pipeline, thus producing reproducible results. You can set it also to any other positive integer to ensure reproducibility. Set the parameter to 0 to use a random seed."
```

Comment:
I don't like that `0` is the value for a random seed, because 0 is also a integer. Why not `null` instead?